src/utils/tracking/README.md

```diff
@@ -0,0 +1,202 @@
+# Google Analytics (GA) Tracking In Jaeger UI
```

Comment:
Great write up Joe

Although I'm all for documentation my only comment would be, is it really worth to maintain all this stuff for Jaeger UI? Although this is nice to have, my concern is this information will go stale as its not really part of the core of what Jaeger provides.

Reply:
That's a valid point. But, it really shouldn't need to change much. Seems like low-maintenance docs, to me. 

What do you suggest?